config/database.php

```diff
@@ -56,7 +56,7 @@
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => '',
             'prefix_indexes' => true,
-            'strict' => true,
+            'strict' => false,
```

Comment:
Why did you change this? Not sure if that's wanted for production?

Reply:
Accidental commit. I have to do this to get it working locally. I will revert.